mavros/src/plugins/sys_status.cpp

```diff
@@ -946,6 +1000,26 @@ class SystemStatusPlugin : public plugin::Plugin
     statustext_pub->publish(st_msg);
   }
 
+  void handle_event(
+    const mavlink::mavlink_message_t * msg [[maybe_unused]],
+    mavlink::common::msg::EVENT & eventm,
+    plugin::filter::SystemAndOk filter [[maybe_unused]])
+  {
+    uint8_t severity = eventm.log_levels & 0x0F; // take only 4 LSB, see https://mavlink.io/en/messages/common.html#EVENT
+    uint32_t px4_id = eventm.id & 0x00FFFFFF; // take only 24 LSB, see PX4 "constexpr uint32_t ID(const char (&name)[N])" function
+    process_event_normal(severity, px4_id, eventm.arguments);
+
+    auto evt_msg = mavros_msgs::msg::StatusEvent();
+    evt_msg.header.stamp = node->now();
+    evt_msg.severity = severity;
+    evt_msg.px4_id = px4_id;
+    evt_msg.event_time_boot_ms = eventm.event_time_boot_ms;
```

Comment:
Usually time boot used to calculate header's time stamp.

Reply:
Done in new commit